Classes/Configuration/ConfigurationFinder.php

```diff
@@ -166,4 +142,11 @@ private static function buildCustomDimensions(array $configurations): array
 
         return $customDimensions;
     }
+
+    private static function getYamlFileLoader(): YamlFileLoader
+    {
+        self::$yamlFileLoader ??= GeneralUtility::makeInstance(YamlFileLoader::class);
```

Comment:
Actually, I shy away from using GeneralUtility::makeInstance() at this point: when the ConfigurationFinder is utilized, there is no dependency injection container available. This might work now (haven't tested it), but won't bet on it for future TYPO3 versions. Better would be to instantiate manually:

    new YamlFileLoader(new NullLogger());

But: as TYPO3 uses here dependency injection already for the logger, this might cause problems in future versions, when other dependencies might be injected. Then this may result in removing this feature and using the plain Symfony YamlFileLoader again.

Another possibility will be to copy the import functionality over to this extension (or the whole class without the logging).

What do you think? @tbal